src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -298,10 +299,16 @@ class WebUIHostImpl(
     val id = decodedJson?.get("id")?.asString
     val arg = decodedJson?.get("arg")?.asString
 
-    if (command == "auth") {
-      val authKind = decodedJson.get("authKind")?.asString
-      if (authKind == "signout") {
-        CodyAuthenticationManager.getInstance(project).setActiveAccount(null)
+    if ((command == "auth" && decodedJson.get("authKind")?.asString == "signout" ||
+        command == "cody.auth.signout")) {
+      CodyAuthenticationManager.getInstance(project).setActiveAccount(null)
+    } else if (isCommand && id == "cody.auth.switchAccount") {
+      runInEdt {
+        ShowSettingsUtil.getInstance().showSettingsDialog(project, AccountConfigurable::class.java)
+      }
+    } else if (isCommand && id == "cody.status-bar.interacted") {
```

Comment:
This is *such* a weird command name for this purpose, but I guess it is what it is...